src/views/homepage/components/chiclets/chiclets.module.css

```diff
@@ -36,7 +40,7 @@
 .navList {
 	display: flex;
 	flex-wrap: wrap;
-	gap: 16px;
+	gap: 15px;
```

Comment:
🗒️ Safari rounds fractional pixels up, and so the chiclets weren't rendering the same as in Chrome. This smaller `gap` helps on the larger viewport sizes.